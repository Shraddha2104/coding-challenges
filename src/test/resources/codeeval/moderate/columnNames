12204
8825
6891
8714
17652
4851
6827
16409
1799
9068
9242
24
3756
11327
4725
6172
12205
10476
425
11254
13026
2
16696
14919
58
4524
13938
11593
11768
10663
6024
11359
7885
2367
7755
10256
3800
178
11374
30
