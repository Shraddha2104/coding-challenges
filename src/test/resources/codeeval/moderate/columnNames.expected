RAJ
MAK
JEA
LWD
ZBX
GDO
JBO
XGC
BQE
MJT
MQL
X
ENL
PSQ
FYS
ICJ
RAK
OLX
PI
PPV
SFZ
B
XRD
VAU
BF
FQZ
TPB
QCW
QJP
OTC
HWR
PTW
KQG
CMA
KLG
ODL
EPD
FV
PUL
AD
